### PR TITLE
Checking in code changes to support OCP 4.12.18 in AWS..

### PR DESCRIPTION
**Pull request by NatarajBTI**
Checking in code changes to support OCP 4.12.18 in AWS..

### CODE REVIEW

File: aws/ocp-terraform/ocs/gp2.yaml
[the remark this answers is not in the document]
Comment (shajeena) [reply aws/ocp-terraform/ocs/gp2.yaml:1]: ssss

File: aws/deploy.sh
[the remark this answers is not in the document]
Comment (shajeena) [reply aws/deploy.sh:1]: its good to have , to add the files in template.tf. As its working now we can go ahead with this for now

File: aws/ocp-terraform/ocs/gp2.yaml
[the remark this answers is not in the document]
Comment (shajeena) [reply aws/ocp-terraform/ocs/gp2.yaml:1]: Cleanup the yaml file , remove the resourceVersion managedfields etc

**Comment by shajeena** [comment]
Changes done . Thanks @NatarajBTI  Approved

**Review by shajeena — APPROVED** [review]

**Comment by NatarajBTI** [comment]
Done changes as per Shajeena's comment..Please review..

**Review by amitmangalvedkar — APPROVED** [review]
Approved!!!!

**Comment by padmankosalaram** [comment]
Noticed 2 SC set as default `gp2` and `gp3-csi`, Can we please set only one as default as I see this alert `MultipleDefaultStorageClasses` below 

image
image

**Review by padmankosalaram — APPROVED** [review]

**Pull request merged**